network.go

```diff
@@ -24,7 +24,15 @@ type netWatcher struct {
 }
 
 func (ca *Cagent) NetWatcher() *netWatcher {
-	return &netWatcher{cagent: ca, constantlyExcludedInterfaceCache: map[string]bool{}}
+	if ca.netWatcher != nil {
+		return ca.netWatcher
+	}
+
+	ca.netWatcher = &netWatcher{
+		cagent: ca,
```

Comment:
actually, it is gofmted and goimported.. Tried multiple times

Reply:
If I save the file, I get a change

Reply:
![screenshot 2018-11-26 at 11 43 43](https://user-images.githubusercontent.com/800562/49009315-8da78f80-f170-11e8-9137-b3a21a05d3c4.png)

Reply:
I got the reason. IDE used Go v1.10.2. Looks like it doesn't add tabs in case there is a big difference. 
When I changed Go version to 1.11.2 it starts to format as you show 😅